process/Release.md

```diff
@@ -1,20 +1,29 @@
 **Scenario:** You want to release a new version
 
-- Add issues to release on JIRA
+- Notify team of release start
+- Set release start date to today
```

Comment:
That's just moreso thinking from my experience -- I'm usually only handling the JIRA thingy. It seems that there are advantages to making it intentionally broad -- i'll make it your call.